back/main.py

```diff
@@ -3,12 +3,19 @@
 from scripts.utils.logger_manager import LoggerManager
 from scripts.workflow.workflow_manager import WorkflowManager
 
+from back.scripts.utils.config import project_config
```

Comment:
Wouldn't this object be similar as a global variable? Hence it should be in capital letter no?

Reply:
As you prefer.